tests/integration/commands/test_deletion.py

```diff
@@ -32,17 +32,6 @@ def test_topic_deletion_without_verification_does_not_work(
     assert topic in topics
 
 
-@pytest.mark.integration
-def test_delete_topic_without_topic_name_fails(
```

Comment:
Why is this test being removed?

Reply:
When you pipe arguments from a different command, the current arguments are empty. So the argument is not required.

Reply:
I refactored it to handle the result differently.